modules/check_in/spec/request/v2/pre_check_in_request_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'V2::PreCheckInController', type: :request do
+  let(:id) { '2f90aa72-deba-4bd7-9bd6-5d17c70bb1b3' }
+  let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store) }
```

Comment:
minor: probably not needed at this point.

Reply:
Sure Gaurav, we might need when we add specs for verifying funcitonality.  I will leave it as it is for now.